scripts/install_dependencies/install_NEO_OCL_driver.sh

```diff
@@ -383,8 +378,8 @@ _check_distro_version()
 {
     if [[ $DISTRO == redhat ]]; then
         RHEL_MINOR_VERSION_SUPPORTED="[3-7]"
-        RHEL_VERSION=$(grep -m1 'VERSION_ID' /etc/os-release | grep -Eo "8.${RHEL_MINOR_VERSION_SUPPORTED}")
-        if [[ $? -ne 0 ]]; then
+        cmd=$(grep -m1 'VERSION_ID' /etc/os-release | grep -Eo "8.${RHEL_MINOR_VERSION_SUPPORTED}")
+        if ! cmd; then
```

Comment:
That command is not working. Should it be replaced with this:
```suggestion
        if [ -z "$cmd" ]; then
```